omegaconf/basecontainer.py

```diff
@@ -534,7 +530,7 @@ def _set_item_impl(self, key: Any, value: Any) -> None:
 
         input_config = isinstance(value, Container)
         target_node_ref = self._get_node(key)
-        special_value = value is None or value == "???"
+        special_value = value is None or _is_missing_value(value)
```

Comment:
Is this fixing a bug? (for instance `DictConfig("???") == "???"` used to be False before)
If yes can a test be added for it?

Reply:
Still seems to be False.
This is supposed to be equivalent.

Reply:
But `_is_missing_value(DictConfig("???"))` is True, so it's not equivalent

Reply:
I think In practice it's equivalent due to the many other layers of logic before and after this.
If you can demonstrate a user facing change from this I am happy to respond to it.

Reply:
Not going to dig deeper myself, just wanted to point it out => ok :)